backend/resources/utils/install_util_not_embeded.go

```diff
@@ -0,0 +1,58 @@
+//go:build docker
+
+package utils
+
+import (
+	"embed"
+	"fmt"
+	"os"
+	"runtime"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+// InstallImpl installs mongoutil, mysqlutil, postgres in resourceDir.
+func InstallImpl(resourceDir, utilDir, tarName, version string, _ embed.FS) error {
+	preloadingDir := getPreloadingUtilDir(utilDir)
+	if _, err := os.Stat(preloadingDir); err != nil {
+		if os.IsNotExist(err) {
+			// source file doesn't exist.
+			return errors.Errorf("preloadingDir %q does not exist", preloadingDir)
+		}
+		return errors.Wrapf(err, "preloadingDir %q error", preloadingDir)
+	}
+	if utilDir == preloadingDir {
```

Comment:
Move shortpath above line 17.

Reply:
It would be better to check if this directory exists ?